packages/serve/src/index.ts

```diff
@@ -28,7 +27,7 @@ export default function serve(...args: string[]): void {
         parsedWebpackArgs.unknownArgs
             .filter((e) => e)
             .forEach((unknown) => {
-                logger.error('Unknown argument:', unknown);
+                console.error('Unknown argument:', unknown);
```

Comment:
Let's keep `webpack-cli/lib/utils/logger` and just return `console`, it encapsulates the logger, and in the future gives the convenience of moving to another